osu.Game/Properties/AssemblyInfo.cs

```diff
@@ -11,3 +11,6 @@
 [assembly: InternalsVisibleTo("osu.Game.Tests.Dynamic")]
 [assembly: InternalsVisibleTo("osu.Game.Tests.iOS")]
 [assembly: InternalsVisibleTo("osu.Game.Tests.Android")]
+
+// intended for Moq usage
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
```

Comment:
what's this one for?

Reply:
It's for the test. The test uses a mock `IOverlayManager`, which is an internal interface.

Alternative would be making it public I guess?

Reply:
It makes sense after googling what `DynamicProxyGenAssembly2` is so I guess it's okay.